tests/testthat/test-Report_MetricTable.R

```diff
@@ -9,16 +9,16 @@ test_that("Correct data structure when proper dataframe is passed", {
     dplyr::filter(MetricID == unique(reportingResults$MetricID)[1])
   result <- Report_MetricTable(reportingResults_filt, reportingGroups)
   expect_s3_class(result, "gt_tbl")
-  expect_true(any(grepl("162", result)))
-  expect_true(any(grepl("Kimler", result)))
+  #expect_true(any(grepl("162", result)))
```

Comment:
with "new" results, I'm not quite matching the hard coded testthat results all the time, might be a result of recreating it incorrectly, but can modify tests to be more data agnostic or hard code new ones?

Reply:
you can modify tests to be more data agnostic